docs/source/explore/get-started/install.rst

```diff
@@ -2,39 +2,61 @@
+Vircadia has two different installers. The *Client Installer* comes with everything you need to view and interact with Vircadia's content and users. However, you are unable to host content using this installer. The *Client + Server Installer* has everything the client installer does, but it also enables you to :doc:`host your own content <../../host>` and share it with the world. *The server sandbox is currently only supported on Windows. Alternatively, you can install a server on Linux.*
 
 .. contents:: On This Page
     :depth: 2
 
---------------------------------
-Minimum System Requirements
---------------------------------
+-------------------
+System Requirements
+-------------------
 
-In order to run Vircadia in either VR or Desktop mode, ensure that your computer meets these minimum system requirements: 
+In order to run Vircadia, ensure that your computer meets these system requirements:
 
-* Windows 10, 64-bit or MacOS High Sierra (10.13)
-* 8GB+ RAM i5
-* A wide range of Nvidia, AMD, and Intel graphics cards are supported
+* Operating system:
+
+  * Windows 8.1, 64-bit or Windows 10, 64-bit
+  * MacOS High Sierra (10.13) or newer
+  * Ubuntu 18.04 64 bit or any other modern Linux distribution
+
+* CPU:
+
+  * **Minimum:** Quad-core
+  * **Recommended:** Quad-core with hyper-threading or better
+
+* System memory:
+
+  * **Minimum:** 8GB
+
+* Graphics adapter:
+
+  * **Minimum:** Nvidia 600 series
+  * **Recommended:** Nvidia 900 series or higher
+  * **Minimum:** AMD Radeon HD 87XX series
+  * **Recommended:** AMD Radeon HD 89XX series or higher
+  * **Minimum:** Intel 5th generation CPU integrated Iris graphics. Using Intel integrated graphics is not recommended.
+
+.. note:: The listed graphics adapters support OpenGL 4.1 and above. This is a requirement for any graphics adapter running Vircadia.
 
 In addition, your network must have enough internet bandwidth to run Vircadia:
 
-* If you are using Interface only to explore the metaverse, then you must have internet speeds of at least **10 Mbps download/2 Mbps** upload.
-* If you are hosting a domain on your Sandbox, you need to add **10 Mbps** upload for each user you want to concurrently visit your domain. 
+* If you are using Interface only to explore the metaverse, then you should have internet speeds of at least **10 Mbit/s download** and **2 Mbit/s upload**.
+* If you are hosting a domain-server on a remote machine or your local computer, you need to add **10 Mbit/s upload** for each user that you want to allow to concurrently visit your domain. You will need only **2 Mbit/s upload** per user if you host your assets on an external web-server rather than your domain-server.
```

Comment:
```suggestion
* If you are hosting a domain server on a remote machine or your local computer, you need to add **10 Mbit/s upload** for each user that you want to allow to concurrently visit your domain. You will need only **2 Mbit/s upload** per user if you host your assets on an external web server rather than your domain server.
```